### ISSUE TITLE
[Feature Request] Documentation about recommended use of .train() and .eval()

### ISSUE THREAD

**Issue by valterschutz** [pytorch/rl#3140]
## Motivation

The ClipPPOLoss documentation says "it is highly recommended to keep your model in eval mode" and there is a [ConsistentDropoutModule](https://docs.pytorch.org/rl/0.7/reference/generated/torchrl.modules.ConsistentDropoutModule.html#torchrl.modules.ConsistentDropoutModule) which makes me think that having dropout in normal nn.Modules is not recommended?

I just happened to stumble across this information in the docs, it is not mentioned in any of the tutorials that I have read.

## Solution

- Add a section in the [Getting started with model optimization](https://docs.pytorch.org/rl/stable/tutorials/getting-started-2.html) page that explains whether .eval() should be used consistently or not.
- Use .eval() consistently in the tutorials, notably the [PPO one](https://docs.pytorch.org/rl/stable/tutorials/coding_ppo.html).

## Checklist

- [X] I have checked that there is no similar issue in the repo (**required**)
